lib/jsCompiler.js

```diff
@@ -13,26 +13,31 @@ const getFunctionArgs = require('./getFunctionArgs')
 module.exports = function (app, callback) {
   const params = app.get('params')
   const appName = app.get('appName')
-  const preprocessor = params.jsCompiler.nodeModule
   const jsPath = app.get('jsPath')
   const jsCompiledOutput = app.get('jsCompiledOutput')
   const usingWhitelist = !!params.jsCompilerWhitelist
   const logger = require('./logger')(app)
+  const fsr = require('./tools/fsr')(app)
+  let preprocessor
   let jsFiles
   let preprocessorModule
   let preprocessorArgs
   let promises = []
 
-  if (params.jsCompiler === 'none') {
+  if (params.jsCompiler === 'none' || params.jsCompiler === null) {
     callback()
     return
   }
 
+  preprocessor = params.jsCompiler.nodeModule
+
   // require preprocessor
   try {
     preprocessorModule = prequire(preprocessor)
   } catch (err) {
-    logger.error(`${appName} failed to include your JS compiler! Please ensure that it is declared properly in your package.json and that it has been properly insalled to node_modules.\n`.red, err)
+    logger.error(`${appName} failed to include your JS compiler! Please ensure that it is declared properly in your package.json and that it has been properly insalled to node_modules.`.red)
```

Comment:
fix "insalled"

Reply:
Change default for JS compiler to "none" to suppress the errors/warnings. The generator can opt-in to preprocessors.